src/apdu_sign.c

```diff
@@ -150,28 +238,59 @@ bool is_standard_lock_script(mol_seg_t *lockScript) {
 }
 
 bool is_dao_type_script(mol_seg_t *typeScript) {
-	static const uint8_t defaultTypeScript[] = { 0x9b, 0xd7, 0xe0, 0x6f, 0x3e, 0xcf, 0x4b, 0xe0, 0xf2, 0xfc, 0xd2, 0x18, 0x8b, 0x23, 0xf1,0xb9,0xfc,0xc8,0x8e,0x5d,0x4b,0x65,0xa8,0x63,0x7b,0x17,0x72,0x3b,0xbd,0xa3,0xcc,0xe8 };
+	static const uint8_t defaultTypeScript[] = {
+		0x82, 0xd7, 0x6d, 0x1b, 0x75, 0xfe, 0x2f, 0xd9, 0xa2, 0x7d, 0xfb, 0xaa, 0x65, 0xa0, 0x39, 0x22, 0x1a, 0x38, 0x0d, 0x76, 0xc9, 0x26, 0xf3, 0x78, 0xd3, 0xf8, 0x1c, 0xf3, 0xe7, 0xe1, 0x3f, 0x2e };
```

Comment:
Is it safe to assume this isn't going to change any time soon?

Reply:
I believe so, yes, especially given that it seems to have special handling at nodes so that transactions involving it are allowed to generate new CKB.